### DISCUSSION TITLE
Advanced Options

### DISCUSSION THREAD

**Discussion by AuthoritiveProtocol** [winsiderss/systeminformer#1781] [Questions / Answers] [answered]
Can a Document providing information about the "Advanced" Tab in the "Options" Menu be released that Describes the Function of Certain Available Features that are not so self-evident? or is there is an existing file/link available already? (Ex. EnableServiceNonPoll, EnableStage2, etcetera. ) Any Information is greatly appreciated.

**Comment by dmex** [comment, marked as answer]
> Can a Document providing information about the "Advanced" Tab in the "Options" Menu be released

The `advanced` options are the existing applications settings that you're already seeing elsewhere in the application. It's just an alternative view of the settings in one window instead of several windows/checkboxes/inputs/controls etc...

> (Ex. EnableServiceNonPoll, EnableStage2, etcetera. )

`EnableServiceNonPoll`... We originally planned to support [NotifyServiceStatusChangeW](https://learn.microsoft.com/en-us/windows/win32/api/winsvc/nf-winsvc-notifyservicestatuschangew) and [SubscribeServiceChangeNotifications](https://learn.microsoft.com/en-us/windows/win32/services/subscribeservicechangenotifications) functions but these functions silently drop notifications for `SERVICE_KERNEL_DRIVER` services.... If you `EnableServiceNonPoll` then you won't get any notifications for drivers.

`EnableStage2`... Is the checkbox `Check images for digital signatures`

![image](https://github.com/winsiderss/systeminformer/assets/1306177/4c672b26-4595-4502-9823-19a0c7448d24)
